README.md

```diff
@@ -361,7 +361,7 @@ imagine --tile-x -w 1024 -h 512 "360 degree equirectangular panorama photograph
 Use depth maps for amazing "translations" of existing images.
 
 ```bash
->> imagine --model SD-2.0-depth --init-image girl_with_a_pearl_earring_large.jpg --init-image-strength 0.05  "professional headshot photo of a woman with a pearl earring" -r 4 -w 1024 -h 1024 --steps 50
+>> imagine --init-image girl_with_a_pearl_earring_large.jpg --init-image-strength 0.05  "professional headshot photo of a woman with a pearl earring" -r 4 -w 1024 -h 1024 --steps 50
```

Comment:
needs to be replaced with a demo showing usage of depth controlnet

Reply:
this is already bad documentation in master so doesn't need to be fixed as part of this pull request